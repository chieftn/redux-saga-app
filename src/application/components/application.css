body {
    margin: 0px;
    overflow: none;
}

.header {
    height: 40px;
    padding: 5px;
    background-color: black;
    color: white;
    margin: 0px;
}

.content {
    height: calc(100vh - 60px);
    overflow-y: auto;
}