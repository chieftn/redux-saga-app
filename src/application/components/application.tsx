import * as React from 'react';
import { Provider } from 'react-redux';
import { Fabric } from 'office-ui-fabric-react/lib/Fabric';
import store from '../../redux/store';
import { DeviceList } from '../../devices/components/deviceListPromise';
import './application.css';

export const Application: React.FC = () => {
    return (
        <Provider store={store}>
            <Fabric>
                <div>
                    <h2 className="header">Devices View</h2>
                    <div className="content">
                        <DeviceList />
                    </div>
                </div>
            </Fabric>
        </Provider>
    );
};
