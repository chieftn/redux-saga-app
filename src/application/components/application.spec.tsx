import * as React from 'react';
import { render } from '@testing-library/react';
import { Application } from './application';

describe('application', () => {
    it('runs test', () => {
        expect(1).toEqual(1);
    });

    it('renders', () => {
        const { container, asFragment } = render(<Application />);
        expect(container).toMatchSnapshot();
    });
});
