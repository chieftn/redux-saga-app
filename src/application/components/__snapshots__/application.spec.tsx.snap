// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`application renders 1`] = `
<div>
  <div>
    Hello World
  </div>
</div>
`;
